src/openrct2/util/SawyerCoding.cpp

```diff
@@ -219,8 +219,9 @@ static size_t DecodeChunkRLEWithSize(const uint8_t* src_buffer, uint8_t* dst_buf
 
     dst = dst_buffer;
 
-    assert(length > 0);
-    assert(dstSize > 0);
+    if (length <= 0 || dstSize <= 0)
+        throw std::runtime_error("Invalid RLE string!");
```

Comment:
Shouldn't it be `std::out_of_range`?

Reply:
Didn’t know it existed ;)

Shall I use that in the other place too?

Reply:
I think so, it's a more accurate description of the error

Reply:
Good, updated it in both places.